### PR TITLE
Fixed TestableLivewire to always return an instance of itself

### LINKED ISSUE #2507: Livewire assertSeeHtml test helper isn't stackable
I want to use this code:

```php
Livewire::test(Create::class, ['resourceSlug' => $resource->slug()])
    ->assertStatus(200)
    ->assertSee('Body')
    ->assertSee('textarea')
    ->assertSeeHtml('rows="3"')
    ->assertDontSeeHtml('rows="6"');
```

But I get this error message:

```
Method Illuminate\Http\Response::assertSeeHtml does not exist.
```

So the `assertSee` helper returns an `Illuminate\Http\Response` instance, so I can't stack `assertSeeHtml` on it.

Fortunately I found a simple workaround:

```php
Livewire::test(Create::class, ['resourceSlug' => $resource->slug()])
    ->assertStatus(200)
    ->assertSee('Body')
    ->assertSee('textarea');

Livewire::test(Create::class, ['resourceSlug' => $resource->slug()])
    ->assertSeeHtml('rows="3"')
    ->assertDontSeeHtml('rows="6"');
```

But it would still be nice if it gets fixed.

Thank you! Jeroen

### PR DESCRIPTION
**Pull request by austenc**
Fixes https://github.com/livewire/livewire/issues/2507 

Having `assertStatus` (or any other assertion not extended by Livewire) and then trying to use Livewire-specific assertions causes an error. This is because `assertStatus` returns an instance of `TestReponse` and not an instance of `TestableLivewire`.

### CODE REVIEW

**Comment by austenc** [comment]
Tests should pass! Not sure what happened here.

File: src/Testing/TestableLivewire.php [hunk 1]
```diff
@@ -247,6 +247,7 @@ public function __call($method, $params)
             return $this->macroCall($method, $params);
         }
 
-        return $this->lastResponse->$method(...$params);
+        $this->lastResponse->$method(...$params);
```
Comment (calebporzio) [inline src/Testing/TestableLivewire.php:250]: Can you add a line break after this and an accompanying test?

Comment (austenc) [reply src/Testing/TestableLivewire.php:250]: Done!

**Review by nuernbergerA — APPROVED** [review]

**Comment by calebporzio** [comment]
Thanks Austen!

**Pull request merged**